packages/website/src/utils/TableHOCExampleUtils.tsx

```diff
@@ -1,9 +1,9 @@
+import {DateUtils, TableRowConnected, TableRowNumberColumn} from '@coveord/plasma-react';
 import moment from 'moment';
 import * as React from 'react';
-import {DateUtils, TableRowConnected, TableRowNumberColumn} from '@coveord/plasma-react';
 
-import {IExampleRowData} from '../routes/layout/pages/TableHOCExamples';
-import {TableHOCServerExampleId} from '../routes/layout/pages/TableHOCServerExamples';
+import {IExampleRowData} from '../pages/layout/TableHOCExamples';
+import {TableHOCServerExampleId} from '../pages/layout/TableHOCServerExamples';
```

Comment:
Even if I prefer not to prefix the folders with an @ when they belong to the project

```suggestion
import {IExampleRowData} from '@pages/layout/TableHOCExamples';
import {TableHOCServerExampleId} from '@pages/layout/TableHOCServerExamples';
```

Reply:
I don't want to use the alias everywhere, its only used for loading tsx files as raw files